src/status_im/contexts/wallet/events.cljs

```diff
@@ -303,62 +304,67 @@
 
 (rf/reg-event-fx :wallet/set-ens-address
  (fn [{:keys [db]} [result ens]]
-   {:db (assoc db
-               :wallet/local-suggestions     (if result
-                                               [{:type     item-types/address
-                                                 :ens      ens
-                                                 :address  (eip55/address->checksum result)
-                                                 :networks [:ethereum :optimism]}]
-                                               [])
-               :wallet/valid-ens-or-address? (boolean result))}))
+   {:db
+    (-> db
+        (assoc-in [:wallet :search-address :local-suggestions]
+                  (if result
+                    [{:type     item-types/address
+                      :ens      ens
+                      :address  (eip55/address->checksum result)
+                      :networks [:ethereum :optimism]}]
+                    []))
+        (assoc-in [:wallet :search-address :valid-ens-or-address?]
+                  (boolean result)))}))
 
 (rf/reg-event-fx :wallet/fetch-address-suggestions
  (fn [{:keys [db]} [_address]]
-   {:db (assoc db
-               :wallet/local-suggestions     nil
-               :wallet/valid-ens-or-address? false)}))
+   {:db (-> db
+            (assoc-in [:wallet :search-address :local-suggestions] nil)
```

Comment:
maybe put it in `:ui, i,e `(assoc-in [:wallet :ui :search-address :local-suggestions] nil)`

Reply:
yeah, good idea!